roles/edpm_network_config/tasks/main.yml

```diff
@@ -41,9 +41,6 @@
         state: restarted
       when: nm_ovs_status.changed  # noqa: no-handler
 
-- name: Import DNS NetworkManager configs tasks
```

Comment:
This task only configures DNS update will be done by NM or not.
is this updating DNS config for nmstate provider when using  cloud-init nmstate config intial setup?